packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -656,8 +674,15 @@ class DescopeWc extends BaseDescopeWc {
         `[${ELEMENT_TYPE_ATTRIBUTE}="polling"]`
       );
       if (loader) {
+        const flowConfig = await this.getFlowConfig();
+        const projectConfig = await this.getProjectConfig();
```

Comment:
I would not let the caller set this here

instead, we can pass it where the function is defined 
(also true for the [change bellow](https://github.com/descope/descope-js/pull/255/files#diff-4e98988585b972daa77cdd3c14af366d7021a2ef1666a77d6b95f50fc68e60c9R768)
![image](https://github.com/descope/descope-js/assets/10514677/b5885c48-a399-467a-828d-991f6d932864))

Reply:
Talked to @nirgur, we prefer to do the types changes after fixing all the tests in the v2 branch